contracts/script/gateway-examples/DeployAndRegisterFunction.s.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+import "forge-std/Vm.sol";
+import "forge-std/console.sol";
+import {FunctionVerifier} from "./FunctionVerifier.sol";
+import {SuccinctGateway} from "../../src/SuccinctGateway.sol";
+
+contract DeployAndRegisterFunction is Script {
+    function run() external {
+        vm.startBroadcast();
+
+        // Get the bytecode of the FunctionVerifier contract.
+        bytes memory bytecode = type(FunctionVerifier).creationCode;
+
+        // SuccinctGateway address
+        address gateway = vm.envAddress("SUCCINCT_GATEWAY");
```

Comment:
nit: UPPER_CASE for these since we generally do that in other scripts

Reply:
`address OWNER = msg.sender` might be more clear, that way they can change this if desired